server/api/CMakeLists.txt

```diff
@@ -178,6 +179,8 @@ target_include_directories(
   "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
   "$<BUILD_INTERFACE:${CMAKE_IRODS_SOURCE_DIR}/plugins/api/include>"
   "$<BUILD_INTERFACE:${CMAKE_IRODS_SOURCE_DIR}/server/core/include>"
+  "$<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/server/genquery2>" # For parser.hpp.
```

Comment:
Use `CMAKE_IRODS_BINARY_DIR` instead of `CMAKE_BINARY_DIR`

Reply:
Handled by @SwooshyCueb's commit.